docs/design/extending-api.md

```diff
@@ -113,18 +113,17 @@ For example, if a user creates:
 
 ```yaml
 metadata:
-  name: cron-tab.example.com
+  name: cron-tab.stable.example.com
 apiVersion: experimental/v1alpha1
 kind: ThirdPartyResource
 description: "A specification of a Pod to run on a cron style schedule"
 versions:
-  - name: stable/v1
-  - name: experimental/v2
+- v1
```

Comment:
then could you search and remove the reference to this field in the doc? Thank you.
```
The APIGroup this version should belong to.
```

We also need to delete that field from types.go, but that can be done after 1.1.